lib/widget/second_page_widgets/second_page_top.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import '../../app_text/second_page_text.dart';
+
+class SecondPageTop extends StatelessWidget {
+  const SecondPageTop({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      fit: StackFit.expand,
+      children: <Widget>[
+        _buildShopPhoto(),
+        _buildShopTitle(),
+        _buildIcons(),
+      ],
+    );
+  }
+}
+
+Widget _buildShopPhoto() {
+  final image = Image.asset('assets/images/warzywa.jpg', fit: BoxFit.fitHeight);
+  return image;
```

Comment:
ten final nie jest potrzebny, wystarczy `return Image.asset('assets/images/warzywa.jpg', fit: BoxFit.fitHeight)`

Reply:
czyli wsm nie musisz tego wyciągać do _build, to tylko jedna linijka